content/docs/getting_started/report.md

```diff
@@ -82,4 +82,14 @@ The report will be downloaded as an excel file.
 
 Standard reports will be translated to the user's language if available when generating the report.
 
-English will be used by default in places where translations in the user's language is unavailable.
\ No newline at end of file
+English will be used by default in places where translations in the user's language is unavailable.
+
+## Standard Reports
+
+Builds 2.3.0 and above will include standard reports by default in all instances of open mSupply. The 2.3.0 build includes the following reports:
```

Comment:
```suggestion
Version 2.3.0+ will include standard reports by default in all instances of Open mSupply:
```

Are these the default reports that will be available?

Reply:
Yes default reports available.

They're part of the OMS repo now